<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegyInputDialog</class>
 <widget class="QDialog" name="SegyInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ProjectViewer/icons.qrc">
    <normaloff>:/icons/images/logo_fs_64x64.png</normaloff>:/icons/images/logo_fs_64x64.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="leFilename"/>
       </item>
       <item>
        <widget class="QPushButton" name="pbBrowse">
         <property name="whatsThis">
          <string comment="abc" extracomment="Click here to browse for a SEGY file."/>
         </property>
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Key</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Minimum</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Maximum</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbIline">
         <property name="text">
          <string>Inline</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lePosIline"/>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="cbSizeIline">
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="leMinIline">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="leMaxIline">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbXline">
         <property name="text">
          <string>Crossline</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lePosXline"/>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="cbSizeXline">
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="leMinXline">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="leMaxXline">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lbCDP">
         <property name="text">
          <string>CDP</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lePosCDP"/>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="cbSizeCDP">
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="leMinCDP">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLineEdit" name="leMaxCDP">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lbOffset">
         <property name="text">
          <string>Offset</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lePosOffset"/>
       </item>
       <item row="4" column="2">
        <widget class="QComboBox" name="cbSizeOffset">
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLineEdit" name="leMinOffset">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QLineEdit" name="leMaxOffset">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lbCDPX">
         <property name="text">
          <string>CDP X-coordinate</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lePosCDPX"/>
       </item>
       <item row="5" column="2">
        <widget class="QComboBox" name="cbSizeCDPX">
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLineEdit" name="leMinCDPX">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QLineEdit" name="leMaxCDPX">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lbCDPY">
         <property name="text">
          <string>CDP Y-coordinate</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lePosCDPY"/>
       </item>
       <item row="6" column="2">
        <widget class="QComboBox" name="cbSizeCDPY">
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLineEdit" name="leMinCDPY">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QLineEdit" name="leMaxCDPY">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="pbEBCDIC">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>View EBCDIC header</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pbBinary">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>View Binary Header</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pbScanTraceHeaders">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Scan Trace Headers</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pbTraceHeader">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>View Trace Header</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btScan">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Scan Geometry</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pbAdvanced">
         <property name="text">
          <string>Advanced</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbOK">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="wdAdvanced" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="gbScalco">
          <property name="title">
           <string>Coordinate Scaler (scalco)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="QRadioButton" name="rbScalcoHeader">
             <property name="text">
              <string>Header</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QRadioButton" name="rbScalcoFixed">
               <property name="text">
                <string>Fixed</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leScalco">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="cbScaleOffset">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Apply to offset</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="gbScalel">
          <property name="title">
           <string>Elevation Scaler (scalel)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QRadioButton" name="rbScalelHeader">
             <property name="text">
              <string>Header</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QRadioButton" name="rbScalelFixed">
               <property name="text">
                <string>Fixed</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leScalel">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="gbFormat">
          <property name="title">
           <string>Bits and Bytes</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Sample Format:</string>
               </property>
               <property name="buddy">
                <cstring>cbFormat</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbFormat"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QRadioButton" name="rbSwapBytes">
             <property name="text">
              <string>Swap Bytes</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>27</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leFilename</tabstop>
  <tabstop>pbBrowse</tabstop>
  <tabstop>lePosIline</tabstop>
  <tabstop>cbSizeIline</tabstop>
  <tabstop>lePosXline</tabstop>
  <tabstop>cbSizeXline</tabstop>
  <tabstop>lePosCDP</tabstop>
  <tabstop>cbSizeCDP</tabstop>
  <tabstop>lePosOffset</tabstop>
  <tabstop>cbSizeOffset</tabstop>
  <tabstop>lePosCDPX</tabstop>
  <tabstop>cbSizeCDPX</tabstop>
  <tabstop>lePosCDPY</tabstop>
  <tabstop>cbSizeCDPY</tabstop>
  <tabstop>pbEBCDIC</tabstop>
  <tabstop>btScan</tabstop>
  <tabstop>leMaxCDP</tabstop>
  <tabstop>leMinIline</tabstop>
  <tabstop>leMaxIline</tabstop>
  <tabstop>leMinCDPY</tabstop>
  <tabstop>leMinCDPX</tabstop>
  <tabstop>leMinXline</tabstop>
  <tabstop>leMaxXline</tabstop>
  <tabstop>leMinCDP</tabstop>
  <tabstop>leMaxCDPY</tabstop>
  <tabstop>leMinOffset</tabstop>
  <tabstop>leMaxOffset</tabstop>
  <tabstop>leMaxCDPX</tabstop>
 </tabstops>
 <resources>
  <include location="../ProjectViewer/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>rbScalelFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>leScalel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>812</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAdvanced</sender>
   <signal>toggled(bool)</signal>
   <receiver>wdAdvanced</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>SegyInputDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>718</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbScalcoFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>leScalco</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
